dockstore-webservice/src/main/java/io/dockstore/webservice/core/Version.java

```diff
@@ -67,6 +69,11 @@
 @Entity
 @ApiModel(value = "Version", description = "Base class for versions of entries in the Dockstore")
 @Inheritance(strategy = InheritanceType.TABLE_PER_CLASS)
+
+@NamedQueries({
+        @NamedQuery(name = "Version.findVersionInEntry", query = "SELECT v FROM Version v WHERE :entryId = v.parent.id AND :versionId = v.id")
```

Comment:
The typical pattern for naming named queries is to use the package name, although we're not always consistent in it (io.dockstore.webservice.core.Version.findVersionInEntry).